list_filtering.py

```diff
@@ -0,0 +1,23 @@
+l = [1, 2, '3', 4, None, 10, 33, 'Python', -37.5]
+
+
+def filtering_integer_by_for():
+    for item in l:
+        if type(item) == int:

```

Comment:
What build-in function can be used for checking whether the specified value is of a specified type? Could you please update this condition using this function?